subsystems/drivebase/SwerveDrive.java

```diff
@@ -225,8 +228,13 @@ public void mustangPeriodic() {
             }
         }
         mPoseEstimator.update();
-        SmartDashboard.putNumber("navX heading", getPose().getRotation().getDegrees());
-        SmartDashboard.putNumber("pitch", getPitch());
+        Logger.getInstance().recordOutput(getName()+"/navX Heading Deg", getPose().getRotation().getDegrees());
+        Logger.getInstance().recordOutput(getName()+"/pitch",getPitch());
```

Comment:
a style thing: there should be a space after the comma. You can set up your editor to just do this for you. We're being inconsistent about this within two lines (see previous line)